x-pack/test_serverless/functional/services/svl_reporting.ts

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { X_ELASTIC_INTERNAL_ORIGIN_REQUEST } from '@kbn/core-http-common';
```

Comment:
This file is kind of a duplicate of x-pack/test_serverless/api_integration/services/svl_reporting.ts. It looks like we want pretty much the same functions for both functional and api_integration. It looks like we can get this just putting our `svl_reporting.ts` file in `x-pack/test_serverless/shared/services/` and expose it in the index.ts with other shared services.

After that, we can back out the changes to `x-pack/test_serverless/functional/services/index.ts` and `x-pack/test_serverless/api_integration/services/index.ts`.